app/src/main/java/viewModel/AccountViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package viewModel
+
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class AccountViewModel : ViewModel() {
+    val titles: MutableLiveData<MutableList<String?>> by lazy {
+        MutableLiveData<MutableList<String?>>(initTitles())
+    }
+
+    private fun initTitles(): MutableList<String?> {
+        Log.d("TAG", "New instance of AccountViewModel created")
+        return mutableListOf(null, "Title 2", "Title 3")
+    }
+
+    fun addTitle(title: String) {
+        titles.value?.add(title)
+        titles.notifyObserver()
```

Comment:
I'm not a fan of this notify observer - the goal of live data is that the observing process is automatic.
The name of this function is **confusing** since the side effect of what you do is notifying the observer.

Please use a name that describes what you which is basically reset the list.